spec/api-browser-window-spec.js

```diff
@@ -2514,6 +2514,15 @@ describe('BrowserWindow module', () => {
     const code = `(() => "${expected}")()`
     const asyncCode = `(() => new Promise(r => setTimeout(() => r("${expected}"), 500)))()`
     const badAsyncCode = `(() => new Promise((r, e) => setTimeout(() => e("${expectedErrorMsg}"), 500)))()`
+    const errorCodes = {
+      Error: 'Promise.reject(new Error("Wamp-wamp"))',
+      ReferenceError: 'Promise.reject(new ReferenceError("Wamp-wamp"))',
+      EvalError: 'Promise.reject(new EvalError("Wamp-wamp"))',
+      RangeError: 'Promise.reject(new RangeError("Wamp-wamp"))',
+      SyntaxError: 'Promise.reject(new SyntaxError("Wamp-wamp"))',
+      TypeError: 'Promise.reject(new TypeError("Wamp-wamp"))',
+      URIError: 'Promise.reject(new URIError("Wamp-wamp"))'
```

Comment:
Maybe calls of `Promise.reject()` should be part of the test, not the list of error types?

```js
const errorTypes = new Set([
  Error,
  ReferenceError,
  <...>
])

<...>

for (const ErrorType of errorTypes) {
  <...>
  ipcRenderer.send('executeJavaScript', `Promise.reject(new ${ErrorType}("Wamp-wamp"))`, true)
  <...>
}
```